packages/conversation/src/rest/v1/messages/messages-api.ts

```diff
@@ -361,6 +361,12 @@ export class MessagesApi extends ConversationDomainApi {
       'Accept': 'application/json',
     };
 
+    // Special fields handling: ttl_seconds: number => ttl: string
+    if (data.sendMessageRequestBody.ttl_seconds !== undefined) {
+      (data as any).sendMessageRequestBody.ttl = data.sendMessageRequestBody.ttl_seconds + 's';
+      delete data.sendMessageRequestBody.ttl_seconds;
```

Comment:
⚠️ Very not fan of this: objects immutability
What  about a user trying to reuse same instance across different calls? 
After 1st serialization, the field is deleted and then following "send request" won't act as expected by user

Reply:
I'll create a copy of the object to make sure I don't mess with it.